src/python/evaluation/xlsx_run_tool.py

```diff
@@ -0,0 +1,170 @@
+                             f'{LanguageVersion.JAVA_11.value}, {LanguageVersion.KOTLIN.value}.')
+
+    parser.add_argument('-tp', '--tool-path',
+                        default=Path('src/python/review/run_tool.py').absolute(),
+                        type=lambda value: Path(value).absolute(),
+                        help='Path to script to run on files.')
+
+    parser.add_argument('-tr', '--traceback',
+                        help='If True, column with the full inspector feedback will be added '
+                             'to the output file with results.',
+                        action='store_true')
+
+    parser.add_argument('-ofp', '--output-folder-path',
+                        help='An absolute path to the folder where file with evaluation results'
+                             'will be stored.'
+                             'Default is the path to a directory, where is the folder with xlsx_file.',
+                        # if None default path will be specified based on xlsx_file_path.
+                        default=None,
+                        type=str)
+
+    parser.add_argument('-ofn', '--output-file-name',
+                        help='Filename for that will be created to store inspection results.'
+                             f'Default is "{EvaluationArgument.RESULT_FILE_NAME_EXT.value}"',
+                        default=f'{EvaluationArgument.RESULT_FILE_NAME_EXT.value}',
+                        type=str)
+
+    parser.add_argument(run_tool_arguments.FORMAT.value.short_name,
+                        run_tool_arguments.FORMAT.value.long_name,
+                        default=OutputFormat.JSON.value,
+                        choices=OutputFormat.values(),
+                        type=str,
+                        help=f'{run_tool_arguments.FORMAT.value.description}'
+                             f'Use this argument when {EvaluationArgument.TRACEBACK.value} argument'
+                             'is enabled argument will not be used otherwise.')
+
+
+def create_dataframe(config) -> Union[int, pd.DataFrame]:
+    report = pd.DataFrame(
+        {
+            ColumnName.LANGUAGE.value: [],
+            ColumnName.CODE.value: [],
+            ColumnName.GRADE.value: [],
+        },
+    )
+
+    if config.traceback:
+        report[EvaluationArgument.TRACEBACK.value] = []
+
+    try:
+        lang_code_dataframe = pd.read_excel(config.xlsx_file_path)
+
+    except FileNotFoundError:
+        logger.error('XLSX-file with the specified name does not exists.')
+        raise FileNotFoundError
+
+    try:
+        for lang, code in zip(lang_code_dataframe[ColumnName.LANG.value],
+                              lang_code_dataframe[ColumnName.CODE.value]):
+
+            with new_temp_dir() as create_temp_dir:
+                temp_dir_path = create_temp_dir
+                lang_extension = LanguageVersion.language_extension()[lang]
```

Comment:
A little comment: please, rename function into `language_to_extension_dict`. And probably you should add a function: `language_by_extension`, which call `language_to_extension_dict` function and return exstention. Ehat do you think?

Reply:
Definitely!